### DISCUSSION TITLE
Existing initiatives

### DISCUSSION THREAD

**Discussion by armantorkzaban** [tcfev/forDem#48] [General]
A list of existing projects. 
Discuss similarities, differences, cooperation possibilities, ways to eliminate waste of resources.

**Comment by lukas-h** [comment]
What if we start to build a Flutter app on top of 

https://github.com/decidim/decidim

or

https://github.com/consul/consul/


and create only start to implement the modules that are not yet implemented by them from scratch and integrate them into those platforms.

**Reply by armantorkzaban** [reply]
If we decide to go that way adding Flutter to them would be an option. 
https://docs.flutter.dev/development/add-to-app
However, some of the @desputinski's ideas (I share them) on the things fürDem could achieve and those platform do not have, necessitates a deeper research. 
the ideas: [1 hour ago](https://github.com/tcfev/fuerDem/discussions/48#discussioncomment-3998329)

**Comment by armantorkzaban** [comment]
This is one of the lists that we made in the past months together. There are more and I ask colleagues to help update. 
---

Other Platforms/Projects
source: https://wechange.de/group/vernetzung-buergerinnenbeteiligung/document/wir-und-die-anderen-plattformen/

1. Consul: Online-platform for participation of citizens
2. Decidim: see consul (https://decidim.org/) 
3. Wiki-Arguments: Simple tool to discuss pros and cons of a political suggestion. see for example: https://wikiarguments.piratenpartei-hessen.de/
4. Mobilizon: Social network to share dates for events (culture, club culture)
5. Librevent: Scrape Facebook-Events - more or less automatically
6. Wechange.de: Social network and cooperation platform for political initiatives, networks and so on.
7. Radar.squat.net: Links-Autonomer Veranstaltungskalender (kann man einfach über ical importieren)
8. Kartevonmorgen.org: It is basically a map that shows all kind of alternative stuff, political stuff, fair-trade-stores, organic stores and so on
9. Mastadon/Friendica/Diaspora: Alternative social netzworks using activitypub as a protocol (technisches Protokoll) / "Fediverse" (Allgemeine Begrifflichkeit für die alternativen sozialen Netzwerke) zueinander kompatibel sind (zumindest Mastadon und Friendica)
10. Alternative Soziale Medien: https://write.as/simone-robutti/supercharge-your-events-using-mobilizon-and-mobilizon-reshare
11. Politikvonunten.org: Demo-Seite, für eine Art von Protest- und Veranstaltungsmelder.
12. Gemeinwohlökonomie
13. Es geht LOS: Direkte Demokratie stärken durch losbasierte Bürgerräte: https://www.esgehtlos.org/
14. Allianz für die Ukraine: Ökosystem für Menschen auf der Flucht bereitstellen: https://alliance4ukraine.org/?utm_source=Masterlist&utm_campaign=231965b50b-EMAIL_CAMPAIGN_2018_05_27_05_58_COPY_01&utm_medium=email&utm_term=0_db3ae0b8fc-231965b50b-156884013
15. Demokratie-Plattform: Plattform mit Online-Inhalten, die das Thema Demokratie betreffen: https://app.demokratie-plattform.de/angebote/
16. DINA International: Digitale Begegnungsplattform der internationalen Jugendarbeit: https://dina.international/ (Betrieb über Wechange)
17. D3 - So geht Digital (Stiftung Bürgermut): Tipps und Fortbildungen zum Thema Digitalisierung und Verein: https://so-geht-digital.de/kategorie/hacks-tipps/
18. Digitale Engagement-Plattformen (nach ehrenamtlichen Tätigkeiten suchen): https://www.deutsche-stiftung-engagement-und-ehrenamt.de/aktuelles/digitale-engagement-plattformen/#toggle-id-1

**Comment by armantorkzaban** [comment]
[Gamaan](https://gamaan.org/) is the first Iranian partner candidate that I suggest we approach as soon as we have the MVP-I roadmap revised and started the new series of development. They need to receive a heads-up before we release the beta version and can be part of the early testers of the system. 
@emadvote @Alhastam, let us discuss this. 

- [ ] add this to kanban

**Comment by armantorkzaban** [comment]
@zanzendegi, @emadvote
please take a look at https://github.com/decidim/decidim & https://github.com/consul/consul and tell us for what functionalities we might be able to borrow code from them.

**Reply by lukas-h** [reply]
As I suggested, don’t borrow code, but get one of these systems to run. So you at least have running system, where users can be onboarded.
Once a version is running, it can be iterated on top of that

**Reply by lukas-h** [reply]
this would speed up the timeline by at least a couple of months

**Reply by armantorkzaban** [reply]
For this we need the opinion of the team. Before that we gonna need a proper **comparative analysis** between our needs (Deliberation, Networking, Participation, transparency and censorship defiance features) and what Decidim and Consul offer. An early stage comparison is made by the team. @desputinski can give us more info about the results. 
Are you ready to organise the effort @lukas-h

**Reply by desputinski** [reply]
Fürdem as it has been planned is supposed to have several different approaches/columns, of which one is basically **"participation of members of public in parliamentary politics and processes"**. So this is one column that also decidim and consul are specializing on. The other important column/approach apart from that is organizing civil society in general in order to mobilize and put pressure on the political elites and governments.

**Reply by armantorkzaban** [reply]
But to what extents these two platform do address the necessities of organizing civil society? Could you please elaborate this column and draw a simple comparison between fürDem's plan and what the two platform offer?

**Reply by desputinski** [reply]
- **Decidem**: https://try.decidim.org/
I think this shows very well, how initiatives can form, how they internally can vote on subjects, also organize "Assemblies" or "Processes". So it looks like a platform for project management of initiatives in civil society, organizing internally and networking 

- **Consul**: https://demo.consulproject.org/
Looks basically like a  voting platform in order to discuss and vote within multiple political debates

I think it would make sense to integrate some of the functionalities maybe. From Decidem maybe the "assembly"-functionality in order to start political processes. And from Consul the voting/discussion functionalities.

In comparison to that, in my point of view, what I would think **fürDem** could to better/differently:
- Integrate political events/action/demonstrations that are already being planned and talking place OUTSIDE fürDem
- Advertise political action, rather than orderly participation processes that work like project management. There is no platform for people in existence, who let's say want to know about everything that is going on in terms of non-governmental politics/protests/political action in the field of education in a certain city or region.
- Accept that civil society is chaotic and take an approach that is more like a heuristic: Make political action bigger, let it take place anywhere, inform all parts of civil society so protest can bi maximized
- Use dates/input from other platforms (scraping from facebook events, squat.net, wechange, kartevonmorgen and automatically integrate them via api). This would have the advantage, that fürDem can start with a lot of content and doesn't have to grow first. In the light of internet monopolies it might be impossible to grow to mainstream size otherwise...
- Being fully compatible to other alternative social networks such as mastadon or mobilizon

**Comment by desputinski** [comment]
Also important for us is the "Fediverse":
https://fediverse.party/
https://de.wikipedia.org/wiki/Fediverse

We need an interoperability to other "alternative" social networks, we can share information with and people can exchange political events and so on.
The Berlin based meeting of fediverse activists takes place every 3rd day of the month via BigBlueButton here: https://lecture.senfcall.de/fre-5l2-rjl-dmb  (I think starting 7 p.m.)

These networks are using activitypub as a protocol. See: https://en.wikipedia.org/wiki/ActivityPub

**Comment by desputinski** [comment]
Also important: Scraping Facebook events. So for example political demonstrations/political action via librevent:
https://github.com/tracking-exposed/librevent

And the re-importing it, as mobilizon ins planning to do: https://mobilizon.org/en/

**Comment by Alhastam** [comment]
Hi
I am agree
Telegram account (Gamaan) : t.me/gamaan_research

Sent from Proton Mail mobile

-------- Original Message --------

**Reply by desputinski** [reply]
What's that telegram group?

**Reply by armantorkzaban** [reply]
This is a polling organisation focused on Iran. According to their website: 
The Group for Analyzing and Measuring Attitudes in IRAN
GAMAAN is an independent, non-profit research foundation registered in the Netherlands that studies the attitudes of Iranians towards various social and political subjects. 
www.gamaan.org

**Comment by Alhastam** [comment]
It is not a telegram group it is official telegram account of Gamaan .

Sent from Proton Mail mobile

-------- Original Message --------

**Comment by osAIran** [comment]
https://pol.is/
https://github.com/compdemocracy/
Polis is a real-time system for gathering, analyzing and understanding what large groups of people think in their own words, enabled by advanced statistics and machine learning.

**Reply by tcf-admin** [reply]
We have a spreadsheet in which we compare features of the platforms we find similar. Would you like to join and work on that?

**Reply by osAIran** [reply]
Sounds good

**Reply by armantorkzaban** [reply]
You have access now. It might look messy. We are sure you will make it better. ;-)

**Comment by armantorkzaban** [comment]
https://democracy.foundation/similar-projects/

**Reply by desputinski** [reply]
Thanks. Interesting this view:
https://www.direct-democracy-navigator.org/locations/berlin-germany/regional

Is something like our https://github.com/tcfev/forDem/issues/6

But The platform seems to be missing these essential features:
1. Collaboration functionalities
2. Interoperability
3. An App

**Comment by ARTE23** [comment]
Hello @armantorkzaban,

This project looks super interesting and I would like to ask you about a possible cooperation with a new and small politcal party in Germany. 

It would be nice, if you tell me how I could reach you :)

Best regards,
Arte

**Reply by armantorkzaban** [reply]
Hi @ARTE23,
Nice to meet you. Let us talk about your idea. 
Write to me here: ghr1qajx@anonaddy.me
